components/PayRuns/PayRunCollapsedContent.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import { currency } from '../../constants/strings';
-import { getNumberWithCommas } from '../../service/helpers';
+import PayRunInvoiceItems from './PayRunInvoiceItems'
 
 const PayRunCollapsedContent = ({ invoice = {} }) => {
   const { invoiceItems } = invoice;
-  let totalToPay = 0;
   if (!invoiceItems) return React.Fragment;
+
+  const invoiceClaimedByHackney = invoiceItems.filter(item => item.claimedBy === 'Hackney');
+  const invoiceClaimedByOther = invoiceItems.filter(item => item.claimedBy !== 'Hackney');

```

Comment:
we don't need this variable, this variable is the same as:     !invoiceClaimedByHackney